webshare/OWNERS

```diff
@@ -0,0 +1 @@
+@mgiuca
```

Comment:
Hmm OK, done.

(This seems to be a recurring problem with Web Share. The new name is inconsistent with most of the other names here, like webusb, but there are some precedents.)